python/its-status/its_status/collector.gnss.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, cfg):
 
         self.thread = threading.Thread(
             target=self.__loop,
-            name="gnss",
+            name="gpsd_client",
```

Comment:
so rename to `collector_gpsd.py`?

Reply:
Yes, I'll rename the file, and use `__name__` rather than hard-code the thread name.

Reply:
In fact, no, I don't think we should rename the file: it is supposed to really be a
collector of _gnss_ data; it just happens that, for now, it only knows how to collect
data from gpsd. In the future, we just need to introduce an new config option that
defines the type of gnss source, in the `gnss` section of the config file, like so:

```
[gnss]
type = gpsd
host = 127.0.0.1
port = 2947
```
or:
```
[gnss]
type = http-vendor-X
url = https://localhost:1234/api/rest/gnss
```

Still, I'll tweak the title to include the module name, with `__name__`; that is
indeed interesting to have.